lib/rest/Twilio.js

```diff
@@ -271,6 +268,40 @@ Twilio.prototype.request = function request(opts) {
   });
 };
 
+/* jshint ignore:start */
+/**
+ * Adds a region and/or edge to a given hostname
+ *
+ * @function getHostname
+ * @memberof Twilio#
+ *
+ * @param {string} hostname - A URI hostname (e.g. api.twilio.com)
+ * @param {string} targetEdge - The targeted edge location (e.g. sydney)
+ * @param {string} targetRegion - The targeted region location (e.g. au1)
+ */
+/* jshint ignore:end */
+Twilio.prototype.getHostname = function getHostname(hostname, targetEdge,
+                                                     targetRegion) {
+  const domain = 'twilio.com';
+  const defaultRegion = 'us1';
+
+  const prefix = hostname.split('.' + domain)[0];
```

Comment:
What happens when the hostname doesn't contain `.twilio.com`?

Reply:
Added a test to account for this case. We now have the same behavior as if the domain was `twilio.com`.